BUILD.bazel

```diff
@@ -24,3 +24,9 @@ label_flag(
     build_setting_default = "//tools/rustfmt:rustfmt.toml",
     visibility = ["//visibility:public"],
 )
+
+alias(
```

Comment:
Why this change?

Reply:
I found it to be more consistent with `//:rustfmt.toml`. Having that setting in a different location than the actual call for running rustfmt like unnecessary burden of knowledge. But I can leave this out if you dislike it.

Reply:
I'm fine with the change but let's do a separate PR for that? Then you can give motivation for the change in the PR description, whereas in this PR it's a distraction.

Reply:
https://github.com/bazelbuild/rules_rust/pull/761